include/sparrow/arrow_interface/arrow_array_schema_utils.hpp

```diff
@@ -21,11 +21,50 @@
 #include <type_traits>
 #include <vector>
 
+#include "sparrow/c_interface.hpp"
 #include "sparrow/utils/memory.hpp"
 #include "sparrow/utils/mp_utils.hpp"
 
 namespace sparrow
 {
+    /**
+     * Release the children and dictionnary of an ArrowArray or ArrowSchema.
+     * 
+     * @tparam T ArrowArray or ArrowSchema
```

Comment:
```suggestion
     * @tparam T `ArrowArray` or `ArrowSchema`
```

Keep in mind that doxygen handles markdown syntax.

Reply:
Done